styles/modules/_notifications.scss

```diff
@@ -0,0 +1,113 @@
+.notifications {
+  h1 {
+    font-size: $tx3b;
+    padding: $pad $pad 0;
+    margin-bottom: 5px;
+  }
+
+  .tabs {
+    padding: 0;
+
+    & > a {
+      padding: $pad $padMd;
+      text-decoration: none;
+      font-size: $tx5;
+      font-weight: bold;
+      border-bottom-style: solid;
+      border-bottom-color: transparent;
+      position: relative;
+      bottom: -2px;
+
+      &.active, &:hover {
+        border-bottom-width: 1px;
+        border-bottom-color: inherit;
+      }
+    }
+  }
+
+  .tabContainer {
+    max-height: 400px;
+    max-height: 250px;
+  }
+
+  .notificationsList {
+    border-width: 0;
+    border-top-width: 1px;
+
+    &.noNotifications {
+      text-align: center;
+
+      .fetcherContainer{
+        position: relative;
+        width: 100%;
+        height: 200px;
+
+        .listFetcher {
+          width: 100%;
+          @include center();
+        }
+      }
+    }
+
+    .listFetcher {
```

Comment:
Why are there two places with rules with the listFetcher class? Is it sometimes not inside .fetcherContainer?